source/2020-02-07-the-ember-times-issue-134.md

```diff
@@ -64,13 +65,20 @@ You can find the design system and code on [GitHub](https://github.com/gossi/hok
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Back to the future with Ember Data packages polyfill 📦🚀](https://github.com/ember-data/babel-plugin-ember-data-packages-polyfill)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+The [ember-data packages API](https://emberjs.github.io/rfcs/0395-ember-data-packages.html) allows you to import objects and methods from smaller packages in the ember-data module [since ember-data 3.11](https://blog.emberjs.com/2019/07/15/ember-3-11-released.html).
+
+There's nothing like a new, shiny API that makes your developer life easier. Even better if you can already use that API in your application
+without any upgrading! Users of ember-data with **version 3.10 and below** get to benefit
+from the new way of importing with the help of the [ember-data packages polyfill](https://github.com/ember-data/babel-plugin-ember-data-packages-polyfill)!
+
+The polyfill ensures that any instances of the Packages API import statements are transformed back to the legacy "DS" EmberData import syntax.
+You can opt-in to using the polyfill in your app, by upgrading your ember-cli-babel dependency to [version 7.14](https://github.com/babel/ember-cli-babel/releases/tag/v7.14.0) or higher for this [new feature](https://github.com/babel/ember-cli-babel/pull/318).
+
+Wanna learn more about how to use the new packages import syntax for ember-data? Be sure to give the [Ember 3.11 release post]((https://blog.emberjs.com/2019/07/15/ember-3-11-released.html) another read
+for practical code examples on how to use it and pop by the [#ember-data channel on the Ember Discord](https://discordapp.com/invite/emberjs) for more questions and discussions.
 
```

Comment:
Let me know what the expectations are around length. I would like us to further *why* the package API changes matter. From the [core team call](https://www.notion.so/2020-2-5-37f5adbe29c14379816f7c3ad1882091) I learned about 2 which we could mention briefly: 
- Using package import syntax lays the groundwork for 🌲 Project Trim🌲 and further runtime optimizations to your application. Project Trim is a key part of making ember-data more modern and open to future asset size reduction. Think "you only pay for what you use". 
- The package paths are real and resolvable. This unlocks power in editors like VS Code because it can use the class definition to look things up.